backend/players/models.py

```diff
@@ -2,19 +2,110 @@
 from django.dispatch import receiver
 from django.contrib.auth.models import User
 from django.db import models
+from django.utils import timezone
 
 
 class Player(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    display_name = models.CharField(max_length=30, unique=True)
-    avatar = models.ImageField(upload_to='avatars/', default='fallback.png', blank=True)
+    user = models.OneToOneField(User, on_delete=models.CASCADE) # Registration information
+    display_name = models.CharField(max_length=30, unique=True) # Name that would be displayed to other players
+    avatar = models.ImageField(upload_to='avatars/', default='fallback.png', blank=True) # Profile picture
+    created_at = models.DateTimeField(default=timezone.now) # Date of account creation
+    updated_at = models.DateTimeField(auto_now=True) # Date of last account update
 
     def __str__(self):
-        return self.display_name
+        return f"Player: {self.display_name}"
 
 # Use signal to remove corresponding user on player deletion
 @receiver(post_delete, sender=Player)
 def delete_user_with_player(sender, instance, **kwargs):
     # Delete the associated user when the player is deleted
     if instance.user:
         instance.user.delete()
+
+
+class Tournament(models.Model):
+    name = models.CharField(max_length=100) # Name of the tournament
+    status = models.CharField(max_length=20, choices=[
+        ('scheduled', 'Scheduled'),
+        ('in_progress', 'In Progress'),
+        ('completed', 'Completed')
+    ], default='scheduled') # Status of the tournament
+    winner = models.ForeignKey(Player,
+                               on_delete=models.SET('Unknown'),
```

Comment:
SET('Unknown') Usage:
In the Tournament and Match models, `on_delete=models.SET('Unknown')` is used for the winner and loser fields. This will set the field to the string 'Unknown' rather than None when the related Player is deleted. This might cause type inconsistencies and potential errors.
Suggestion: Consider using `models.SET_NULL` instead and ensuring the field is nullable:

```suggestion
                               on_delete=models.SET_NULL,
```